app/src/test/java/org/pesmypetcare/mypetcare/controllers/community/TestTrUnlikePost.java

```diff
@@ -0,0 +1,67 @@
+package org.pesmypetcare.mypetcare.controllers.community;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.TrUnlikePost;
+import org.pesmypetcare.mypetcare.features.community.forums.Forum;
+import org.pesmypetcare.mypetcare.features.community.groups.Group;
+import org.pesmypetcare.mypetcare.features.community.posts.NotLikedPostException;
+import org.pesmypetcare.mypetcare.features.community.posts.Post;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubCommunityService;
+import org.pesmypetcare.usermanager.datacontainers.DateTime;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Albert Pinto
+ */
+public class TestTrUnlikePost {
+    private User user;
+    private Post post;
+    private TrUnlikePost trUnlikePost;
+
+    @Before
+    public void setUp() {
+        user = new User("John Smith", "johndoe@gmail.com", "1234");
+        Group group = new Group("Husky", "John Doe", DateTime.Builder.buildDateString("2020-04-15"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "John Doe" appears 3 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5443034)